mkosi/__init__.py

```diff
@@ -5368,6 +5383,16 @@ def create_parser() -> ArgumentParserMkosi:
         default=0,
         help="Wait up to SECONDS seconds for the SSH connection to be available when using 'mkosi ssh'",
     )
+    group.add_argument(
+        "--ssh-agent",
+        type=parse_ssh_agent,
+        default=False,
+        metavar="PATH",
+        help="If True, 'mkosi build' will use $SSH_AUTH_SOCK (hint: use sudo -E) to talk to the ssh "
+        "agent to fetch the available public keys when building an image. `mkosi ssh` will pass the "
+        "variable through to ssh when logging in. This option can also be used to manually specify a "
+        "path to the ssh agent socket.",
```

Comment:
Hmm, we generally keep the texts in `--help` fairly short. This is supposed to be reminder, not an explanation…
Why not just say "Path to the ssh agent socket, or true to use $SSH_AUTH_SOCK" ?

This is not markdown, so single quotes should be used, not backticks.